core/src/main/scala/stryker4s/config/Config.scala

```diff
@@ -11,8 +11,9 @@ case class Config(mutate: Seq[String] = Seq("**/main/scala/**/*.scala"),
                   reporters: List[MutantRunReporter] = List(new ConsoleReporter),
                   logLevel: Level = Level.INFO,
                   files: Option[Seq[String]] = None,
-                  excludedMutations: Set[String] = Set.empty,
-                  thresholds: Thresholds = Thresholds()) {
+                  excludedMutations: ExcludedMutations = ExcludedMutations(),
+                  thresholds: Thresholds = Thresholds(),
+                  systemProperties: Set[String] = Set.empty) {
```

Comment:
I don't think a set is the correct list to use. certainly when you are doing this `s"-D$key=$value"` it's most certainly possible to have multiple keys that would have the same value we don't know this. So I would assume duplicates will be very much possible.

So either expect full `["-Dconfig=value"]` or allow duplicates

Reply:
I came up with another argument this morning.

Some properties are not key value, for instance, booleans are not mandatory to be key value pairs.

Reply:
oooooww yes then we misunderstanded for sure